syntax = "proto2";

message s2c_del_player_item_notify {
	required uint32 id = 1;
	required uint32 delta = 2;
	required uint32 count = 3;
}